Sources/ProjectDescription/RunActionOptions.swift

```diff
@@ -24,14 +27,18 @@ public struct RunActionOptions: Equatable, Codable {
     ///
     ///     - simulatedLocation: The simulated GPS location to use when running the app.
     ///     Please note that the `.custom(gpxPath:)` case must refer to a valid GPX file in your project's resources.
+    ///
+    ///     - enableGPUFrameCaptureMode: The capture mode to use. e.g: .disabled
```

Comment:
Hi @PierreCapo, I think you can make `The capture mode to use` a bit more clear. As it is, I'm not sure I would understand it and I think it is quite an obscure options for most iOS dev.

Here is a suggestion:

```suggestion
    ///     - enableGPUFrameCaptureMode: The Metal Frame Capture mode to use. e.g: .disabled
    ///     If your target links to the Metal framework, Xcode enables GPU Frame Capture.
    ///     You can disable it to test your app in best perfomance.
```

Source https://developer.apple.com/documentation/metal/debugging_tools/enabling_frame_capture